deploy/kubernetes/charts/ethereum/values.yaml

```diff
@@ -0,0 +1,62 @@
+replicaCount: 1
+
+image:
+  repository: serai-dev-ethereum
+  pullPolicy: IfNotPresent
+  tag: "latest"
+
+  ports:
+    - name: rpc
+      containerPort: 8545
+      protocol: TCP
+  volumeMounts:
+    - mountPath: /scripts
+      name: configmap-volume
+  args:
+    - bash 
+    - /scripts/entry-dev.sh
+
+volumes:
+  - configMap:
+      defaultMode: 420
+      name: configmap
+    name: configmap-volume
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
```

Comment:
do we need these values if they're empty?

Reply:
deleted unnecessary ones but some of them are needed in next iterations; therefore I had to keep some of them in there.